src/variables/UserID.ts

```diff
@@ -0,0 +1 @@
+export const USER_ID = 102;
```

Comment:
it looks like you create a file only for one constant and called it `USER_ID.ts`

this file sholud have a name like `commonConsts.ts` for example